web/static/js/components/respondents/RespondentRow.jsx

```diff
@@ -30,23 +30,23 @@ class RespondentRow extends Component<Props> {
     return (
       <tr key={respondent.id}>
         <td> {respondent.phoneNumber}</td>
+        <td>
+          {capitalize(respondent.disposition)}
+        </td>
+        <td className='tdDate'>
+          {respondent.date ? dateformat(new Date(respondent.date), 'mmm d, yyyy HH:MM') : '-'}
+        </td>
+        {this.getModeAttemptsValues()}
         {responses.map((response) => {
           // For the 'language' variable we convert the code to the native name
           let value = response.value
           if (response.name == 'language') {
             value = languageNames[value] || value
           }
 
-          return <td className='tdNowrap' key={parseInt(respondent.id) + response.name}>{value}</td>
+          return <td className={`tdNowrap${response.isNumeric ? ' tdNumber' : ''}`} key={parseInt(respondent.id) + response.name}>{value}</td>
```

Comment:
```suggestion
          return <td className={classNames({'tdNowrap' : true, 'tdNumber' : response.isNumeric})} key={parseInt(respondent.id) + response.name}>{value}</td>
```

if `cellClassNames` is added:

```suggestion
          return <td className={cellClassNames(response.field)} key={parseInt(respondent.id) + response.name}>{value}</td>
```